### ISSUE TITLE
Drivers lacking documentation

### ISSUE THREAD

**Issue by Whyborn** [payu-org/payu#470]
The respective drivers lack documentation, requiring assumed knowledge to configure specific models. Require users to contribute an example experiment for the commonly used drivers to demonstrate features specific to that driver?

**Comment by aidanheerdegen** [comment]
Agreed. It would be good to have a first cab off the rank to serve as an exemplar before attempting this for other drivers/models.

Any volunteers? I could have a crack at MOM5 in the first instance, but I really don't know when I could get around to it.
